glActiveTexture error when gl.Init()

panic: glActiveTexture

goroutine 1 [running, locked to thread]:
main.main()
	(my project path)/(my go file name).go:16 +0x50

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	c:/go/src/runtime/asm_386.s:1662 +0x1

I'm using Windows 10, Intel HD4000 Graphic card.
I don't know why this problem occurs.

--- a/README.md
+++ b/README.md
@@ -29,7 +29,7 @@ func main() {
 	window.MakeContextCurrent()
 
 	// Important! Call gl.Init only under the presence of an active OpenGL context,
-	// i.e. after MakeContextCurrent.
+	// i.e., after MakeContextCurrent.
 	if err := gl.Init(); err != nil {
 		panic(err)
 	}

--- a/README.md
+++ b/README.md
@@ -25,10 +25,11 @@ Once the bindings are installed you can use them with the appropriate import sta
 import "github.com/go-gl/gl/v3.3-core/gl"
 
 func main() {
-  window.MakeContextCurrent()
+	window := ... // Open a window.
+	window.MakeContextCurrent()
 
-  // Important! Call gl.Init only under the presence of an active OpenGL context,
-  // i.e. after MakeContextCurrent.
+	// Important! Call gl.Init only under the presence of an active OpenGL context,
+	// i.e. after MakeContextCurrent.
 	if err := gl.Init(); err != nil {
 		panic(err)
 	}

--- a/README.md
+++ b/README.md
@@ -25,6 +25,10 @@ Once the bindings are installed you can use them with the appropriate import sta
 import "github.com/go-gl/gl/v3.3-core/gl"
 
 func main() {
+  window.MakeContextCurrent()
+
+  // Important! Call gl.Init only under the presence of an active OpenGL context,
+  // i.e. after MakeContextCurrent.
 	if err := gl.Init(); err != nil {
 		panic(err)
 	}

